client/my-sites/hosting/sftp-card/index.js

```diff
@@ -309,6 +315,7 @@ export const SftpCard = ( {
 				</FormFieldset>
 			) }
 			{ isLoading && <Spinner /> }
+			{ hasSftpFeatureAndIsLoading && <SshKeyLoadingSkeleton /> }
```

Comment:
Skeleton seems to pulse indefinitely if hosting features aren't yet active:

<img width="1438" alt="image" src="https://user-images.githubusercontent.com/36432/222417947-74f20337-b05b-4e18-8107-7bea7a16e931.png">

I would expect that I'd see the default content right away.

Reply:
Fixed ! https://github.com/Automattic/wp-calypso/pull/73982/commits/4b0d82c7d2776b7fc1a6938d48a788c332baed06